### DISCUSSION TITLE
Thank you PGroonga Team!  www.kelteu.com

### DISCUSSION THREAD

**Discussion by ghevge** [pgroonga/pgroonga#635] [General]
Hey PGroonga Team!

Just wanted to Thank You Guys, and especially @kou   for the quick answers, investigating and fixing the problems that I've reported over the past ~2 years.

I'm glad to announce that the project I was working on in these past 2 years is now public. 
It is available at https://www.kelteu.com. Also available for Android and iOS.

Kelteu.com is a wikipedia like, community driven, application, focused on products and shopping in general. Through this app, you can find details about products: country of origin, contents, ratings, prices + taxes at different geo locations, create and share with friends shopping lists, find geo locations for the products you are looking for, see price histories at different geo locations, and much more.

The application is currently available in 51 languages, and all the search capabilities are driven by PGroonga. 

Thank you PGroonga Team!

PS: Note that most of the geolocated data currently available on the website is from North America.

**Comment by kou** [comment]
Congratulations!

Let's add the site to https://pgroonga.github.io/users/ !
Could you open a pull request that adds the site to https://github.com/pgroonga/pgroonga.github.io/blob/master/users/index.md ?

**Comment by ghevge** [comment]
@kou  https://github.com/pgroonga/pgroonga.github.io/pull/398
